wkt_to_dicomseg/main.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "737ff207-2554-4015-9377-dc898e536585",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import time\n",
+    "import json\n",
+    "from PIL import Image, ImageDraw\n",
+    "import numpy as np\n",
+    "import pydicom\n",
+    "from highdicom.content import (\n",
+    "    AlgorithmIdentificationSequence,\n",
+    "    CodedConcept\n",
+    ")\n",
+    "import highdicom as hd\n",
+    "from shapely import wkt\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "88ca7877-c5d7-4e29-8402-03a36b10bb50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define variables\n",
+    "wkt_input = \"example/input/wkt.json\"\n",
+    "dicom_input = \"example/input/dicom_image.dcm\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c634273c-8a0c-412a-ada8-5dd8096966d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_mask_from_geometry(geometry_data, img_width, img_height):\n",
+    "    \"\"\"Create a binary mask from geometry data.\"\"\"\n",
+    "    # Create blank canvas\n",
+    "    canvas = np.zeros((img_height, img_width))\n",
+    "    mask_image = Image.fromarray(np.uint8(canvas))\n",
+    "\n",
+    "    # Get the mask polygon\n",
+    "    mask_polygon = wkt.loads(geometry_data['wkt'])\n",
+    "\n",
+    "    # Since coordinates seem to be in the correct range already, let's skip scaling\n",
+    "    mask_poly_coords = list(mask_polygon.exterior.coords)\n",
+    "\n",
+    "    if mask_polygon.is_empty:\n",
+    "        return np.zeros((img_height, img_width), dtype=np.uint8)\n",
+    "\n",
+    "    ImageDraw.Draw(mask_image).polygon(mask_poly_coords, fill=1)\n",
+    "\n",
+    "    # Convert to numpy array and check contents\n",
+    "    mask_array = np.array(mask_image, dtype=np.uint8)\n",
+    "\n",
+    "    return mask_array"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "9a92f458-7e35-4d74-9adb-1d695ab2e2aa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_dicom_seg(geometry_data, dicom_file, output_path):\n",
+    "    \"\"\"Create a DICOM SEG file.\"\"\"\n",
+    "    try:\n",
+    "        dicom_dataset = pydicom.dcmread(dicom_file)\n",
+    "        img_width, img_height = dicom_dataset.Columns, dicom_dataset.Rows\n",
+    "\n",
+    "        # Generate masks\n",
+    "        all_masks = []\n",
+    "        segment_descriptions = []\n",
+    "\n",
+    "        for idx, geometry in enumerate(geometry_data, start=1):\n",
```

Comment:
All of our APIs include 0-indexed messaging regarding inputs. I'm not sure we should change that approach here.